pkgs/by-name/as/astroterm/package.nix

```
@@ -0,0 +1,72 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  fetchFromGitHub,
+  xxd,
+  meson,
+  ninja,
+  ncurses,
+  argtable,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "astroterm";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "da-luce";
+    repo = pname;
```

```suggestion
    repo = "astroterm";
```

Avoid interpolating `pname`.